yt/visualization/particle_plots.py

```diff
@@ -243,15 +349,26 @@ def __init__(
         data_source=None,
         deposition="ngp",
         density=False,
+        *,
+        north_vector=None,
+        axis=None,
     ):
+        if axis is not None:
+            issue_deprecation_warning(
+                "The 'axis' argument is a deprecated alias for the 'normal' argument. ",
+                "4.2.0",
+                "4.3.0",
```

Comment:
We don't need to try and predict when we'll be able to end deprecation cycles any more :)
```suggestion
                since="4.2.0",
```

Reply:
(originally wrote this as a nit, but note that `since` and `removal` arguments are actually keyword-only)